bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_rooabspdf.py

```diff
@@ -10,45 +10,34 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.                    #
 ################################################################################
 
-r"""
-/**
-\class RooAbsPdf
-\brief \parblock \endparblock
-\htmlonly
-<div class="pyrootbox">
-\endhtmlonly
-
-## PyROOT
-
-Some member functions of RooAbsPdf that take a RooCmdArg as argument also support keyword arguments.
-So far, this applies to RooAbsPdf::fitTo, RooAbsPdf::plotOn, RooAbsPdf::generate, RooAbsPdf::paramOn, RooAbsPdf::createCdf,
-RooAbsPdf::generateBinned, RooAbsPdf::createChi2, RooAbsPdf::prepareMultiGen and RooAbsPdf::createNLL.
-For example, the following code is equivalent in PyROOT:
-\code{.py}
-# Directly passing a RooCmdArg:
-pdf.fitTo(data, ROOT.RooFit.Range("r1"))
-
-# With keyword arguments:
-pdf.fitTo(data, Range="r1")
-\endcode
-
-\htmlonly
-</div>
-\endhtmlonly
-*/
-"""
 
 from ._rooabsreal import RooAbsReal
 from ._utils import _kwargs_to_roocmdargs
 
 
 class RooAbsPdf(RooAbsReal):
+
+    _doxygen = """Some member functions of RooAbsPdf that take a RooCmdArg as argument also support keyword arguments.
```

Comment:
Why not doing this using the docstring of the class / method? I.e. instead of creating a new attribute called `_doxygen`, you can just create the docstring and access it via `__doc__`

Reply:
I first wanted to keep it separate, because I thought maybe we want to use the docstring for some actual Python documentation is the future, but you're right it's nicer to use the `__doc__` for the PyROOT blocks for now.